bluemira/geometry/coordinates.py

```diff
@@ -924,6 +936,23 @@ def vector_intersect(
     Returns
     -------
     The point of the intersection between the two vectors (shape: (2,)).
+
+    Notes
+    -----
+    If the vectors are parallel:
+        - The vectors do not intersect.
+        - The function returns p2
+
+    Otherwise:
+        - Calculates the intersection point using vector algebra:
+
+        .. math::
+
+            \\text{point} = \\frac{
+                \\mathbf{dap} \\cdot (\\mathbf{p1} - \\mathbf{p3})
```

Comment:
As we're normalising the vector and dap isnt a known quantity from the docstring could I suggest this (if lVert does render use \\|\\|)
```suggestion
                \\lVert \\mathbf{p2} - \\mathbf{p1} \\rVert \\cdot (\\mathbf{p1} - \\mathbf{p3})
```